source/agora/node/Registry.d

```diff
@@ -851,7 +854,7 @@ private struct ZoneData
 
         foreach (const ref payload; this)
         {
-            reply.answers ~= payload.toRR(Domain(format("%s.%s",
+            reply.answers ~= payload.toRR(Domain.fromString(format("%s.%s",
```

Comment:
This can also be `fromSafeString` for consistency with L#619.
PublicKey validates itself and `this.root` is already a `Domain`